apps/rush-lib/src/cli/utilities/test/lockstepRepo/rush.json

```diff
@@ -1,7 +1,7 @@
 {
   "npmVersion": "3.10.8",
   "rushVersion": "1.0.5",
-  "nodeSupportedVersionRange": ">=4.3.0 <7.0.0",
+  "nodeSupportedVersionRange": ">=4.3.0 <9.0.0",
```

Comment:
>"nodeSupportedVersionRange": ">=4.3.0 <9.0.0", [](start = 2, length = 46)

Unless at least one test is validating that failure happens, maybe it's not useful to have this in the test at all (?)

Reply:
I am fine with removing the line if it is optional.

---
In reply to: [160557228](https://github.com/Microsoft/web-build-tools/pull/474#discussion_r160557228) [](ancestors = 160557228)